addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/handler/KodiHandler.java

```diff
@@ -156,19 +163,15 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 break;
             case CHANNEL_PVR_OPEN_TV:
                 if (command instanceof StringType) {
-                    KodiChannelConfig config = getThing().getChannel(channelUID.getId()).getConfiguration()
-                            .as(KodiChannelConfig.class);
-                    playPVRChannel(command, "tv", config);
+                    playPVRChannel(command, "tv", CHANNEL_PVR_OPEN_TV);
```

Comment:
I realize it was like this already, but should `tv` and `radio` be defined as constants, as they seem to be used elsewhere in the code?